packages/nextjs/utils/scaffold-eth/notification.tsx

```diff
@@ -31,7 +31,7 @@ const ENUM_STATUSES = {
 };
 
 const DEFAULT_DURATION = 3000;
-const DEFAULT_POSITION: ToastPosition = "top-center";
+const DEFAULT_POSITION: ToastPosition = "bottom-left";
```

Comment:
This was just a personal preference but happy to change it if people have other opinions